i18n/vscode-language-pack-zh-hans/translations/main.i18n.json

```diff
@@ -509,12 +509,12 @@
 			"miInsertCursorAtEndOfEachLineSelected": "在行尾添加光标(&&U)",
 			"mutlicursor.addCursorsToBottom": "在底部添加光标",
 			"mutlicursor.addCursorsToTop": "在顶部添加光标",
-			"addSelectionToNextFindMatch": "将选择添加到下一个查找匹配项",
+			"addSelectionToNextFindMatch": "将下一个查找匹配项添加到选择",
```

Comment:
My translation looks weird based on the source, but the product does exactly like this, trust me.